java/core/src/main/java/co/worklytics/psoxy/gateway/impl/output/OutputUtils.java

```diff
@@ -27,20 +21,6 @@
 @Singleton
 public class OutputUtils {
```

Comment:
🤔  this is almost more of a factory now; all the metadata stuff moved out, bc it's pretty generic.